src/bin/pgcopydb/ld_transform.c

```diff
@@ -1928,7 +1928,9 @@ stream_write_insert(FILE *out, LogicalMessageInsert *insert)
 
 		for (int c = 0; c < stmt->cols; c++)
 		{
-			appendPQExpBuffer(buf, "%s\"%s\"",
+			const char *quoteFormatStr = (stmt->columns[c][0] == '"') ? "%s%s" :
+										 "%s\"%s\"";
+			appendPQExpBuffer(buf, quoteFormatStr,
```

Comment:
Thanks for the changes! I think it's worth adding a comment about why this is happening...